ports/cuda/vcpkg-cmake-wrapper.cmake

```
@@ -0,0 +1,7 @@
+set(CUDA_TOOLKIT_ROOT_DIR "${CMAKE_CURRENT_LIST_DIR}/../../tools/cuda")

```

```suggestion
set(CUDA_TOOLKIT_ROOT_DIR "${CMAKE_CURRENT_LIST_DIR}/../../tools/cuda" CACHE "FILEPATH" "" FORCE)
```
needs to be a cache variable for libtorchs setup.py to be able to read the variable from the cache